### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Iontr** [TorannD/TMagic#5]
I have a bug when a Elemental Rift open, It does not spawn any Elementals...

Here's the log from HugsLib:
https://git.io/fxgzF

Thanks ;)

@TorannD

**Comment by TorannD** [comment]
It's not really a bug.  What's happened here is that your game is no longer detecting an elemental faction.  If the spawn sees (for whatever reason) that the faction assigned to the elemental is no longer hostile to you, it will force hostility, but when it tries to do so by forcing the faction to elementals and adjusting the hostility level, it's not finding the elemental faction at all and generates the custom error you see.  This can happen if you have another mod that removes or adjusts factions (like hidden factions).

Faction Control mod can and will do this if you do not allow at least 1 summoned/elemental faction. I glanced through your mod list but didn't see this mod or another mod that I know can remove factions. Regardless, the elemental/summoned faction has been removed in your game causing the elemental rift to have no faction to assign to the spawned elementals.
